shared/teams/channel/index.tsx

```diff
@@ -193,11 +193,9 @@ const Channel = (props: OwnProps) => {
     <Kb.Box style={styles.container}>
       {Styles.isMobile && <MobileHeader />}
       <Kb.SectionList
-        alwaysVounceVertical={false}
```

Comment:
?

Reply:
`alwaysVounceVertical` is not a real prop, it's `alwaysBounceVertical` but if it's been the wrong one for sufficiently long and we haven't noticed, we probably don't actually want it.